src/tiled/newtilesetdialog.cpp

```diff
@@ -62,6 +63,8 @@ NewTilesetDialog::NewTilesetDialog(QWidget *parent) :
     mNameWasEdited(false)
 {
     mUi->setupUi(this);
+    mPopup = new ImageColorPickerWidget(mUi->dropperButton);
+    mPopup->setWindowFlags(Qt::Popup);
```

Comment:
Since the `ImageColorPickerWidget` is always meant to be used as a popup, please move the setting of this window flag there. Also, please construct it on-demand in the `pickColorFromImage` function.